src/e2e-test/java/io/cdap/cdap/ui/utils/Helper.java

```diff
@@ -87,6 +97,15 @@ public static WebElement locateElementById(String elementId) {
             .findElement(By.id(elementId));
   }
 
+  public static WebElement locateElementByXPath(String xpath) {
+    return SeleniumDriver.getDriver()
+      .findElement(By.xpath(xpath));
+  }
+
+  public static int getElementCounts(By by) {
```

Comment:
There's already a helper for this here:
https://github.com/cdapio/cdap-e2e-tests/blob/950b748cb27ca9569ff850ded292b6b990c25494/src/main/java/io/cdap/e2e/utils/ElementHelper.java#L405